src/crispy/StrongLRUCache.h

```diff
@@ -22,6 +22,8 @@
 #include <stdexcept>
 #include <vector>
 
+#include "StrongHash.h"
```

Comment:
```suggestion
#include <crispy/StrongHash.h>
```

looks like we need to talk about "" vs <>. I'm generally open for change :)